cmake/modules/AddSwift.cmake

```diff
@@ -997,7 +1021,7 @@ function(_add_swift_library_single target name)
     set_target_properties("${target}"
       PROPERTIES
       INSTALL_NAME_DIR "${install_name_dir}")
-  elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+  elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux" AND NOT "${SWIFTLIB_SINGLE_SDK}" STREQUAL "ANDROID")
```

Comment:
Yes, in my use case it suffices to just run `System.loadLibrary("mySwiftLibrary")` at an appropriately early point in the app's lifecycle - the runtime linker finds the rest of the shared objects provided they're also in the APK bundle. Given that Android is basically a Java-first/-only platform (except for command line tools on rooted devices), this is the "best practices" approach as far as I can tell.